contrib/openshift/README.md

```diff
@@ -63,6 +71,14 @@ Perform the Following steps to access skydive UI
  - Use username = `admin` , Password = `password`  
  - press **sign-in**  
 
+Perform the Following steps to access monitoring data in prometheus
+ - Point your browser to prometheus end-point  
+ `oc get route prometheus -o jsonpath='http://{.spec.host}'`
+ - Go to the popdown menu Status/Targets
+ - Verify that you see the http://skydive-prometheus-connector:9100/metrics endpoint
```

Comment:
This worked for me and I was able to see in the UI that `http://skydive-prometheus-connector:9100/metrics` exists and the state is "UP"